examples/webgl_materials_texture3d_volume1.html

```diff
@@ -0,0 +1,305 @@
+			margin: 0px;
+			overflow: hidden;
+		}
+		#info {
+			color: #fff;
+			position: absolute;
+			top: 10px;
+			width: 100%;
+			text-align: center;
+			z-index: 5;
+			display:block;
+		}
+		.dg {
+			z-index: 10 !important;
+		}
+		#info a, .button { color: #f00; font-weight: bold; text-decoration: underline; cursor: pointer }
+		#inset  {
+			width: 150px;
+			height: 150px;
+			background-color: transparent; /* or transparent; will show through only if renderer alpha: true */
+			border: none; /* or none; */
+			margin: 0;
+			padding: 0px;
+			position: absolute;
+			left: 20px;
+			bottom: 20px;
+			z-index: 100;
+		}
+	</style>
+</head>
+
+<body>
+	<div id="info">
+		<a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> -
+		Float volume render test (mip / isosurface)
+	</div>
+	<div id="inset"></div>
+
+	<script src="../build/three.js"></script>
+
+	<script src="js/controls/OrthographicTrackballControls.js"></script>
+
+	<script src="js/Volume.js"></script>
+	<script src="js/VolumeSlice.js"></script>
+	<script src="js/loaders/NRRDLoader.js"></script>
+	<script src="js/shaders/VolumeShader.js"></script>
+
+	<script src="js/Detector.js"></script>
+	<script src="js/libs/stats.min.js"></script>
+	<script src="js/libs/gunzip.min.js"></script>
+	<script src="js/libs/dat.gui.min.js"></script>
+
+	<script>
+
+		if ( ! Detector.webgl ) Detector.addGetWebGLMessage();
+
+		var container,
+			stats,
+			camera,
+			controls,
+			scene,
+			renderer,
+			gui,
+			container2,
+			renderer2,
+			camera2,
+			axes2,
+			scene2;
+
+		init();
+		animate();
+
+		function init() {
+
+			// The volume renderer does not work very well with perspective yet
+			//camera = new THREE.PerspectiveCamera( 60, window.innerWidth / window.innerHeight, 0.01, 1e10 );
+			camera = new THREE.OrthographicCamera()
+			camera.position.z = 1000;
+			this.camera.right = window.innerWidth / 5.0;
+			this.camera.left = -this.camera.right;
+			this.camera.top = window.innerHeight / 5.0;
+			this.camera.bottom = -this.camera.top;
+			this.camera.near = 0.1;
+			this.camera.far = 5000;
+
+			this.camera.updateProjectionMatrix();
+
+			scene = new THREE.Scene();
+
+			scene.add( camera );
+
+			// light
+
+			var dirLight = new THREE.DirectionalLight( 0xffffff );
+			dirLight.position.set( 200, 200, 1000 ).normalize();
+
+			camera.add( dirLight );
+			camera.add( dirLight.target );
+
+			var loader = new THREE.NRRDLoader();
+			loader.load( "models/nrrd/stent.nrrd", function ( volume ) {
+
+				// Texture to hold the volume. We have scalars, so we put our data in the red channel.
+				// THREEJS will select R32F (33326) based on the RedFormat and FloatType.
+				// Also see https://www.khronos.org/registry/webgl/specs/latest/2.0/#TEXTURE_TYPES_FORMATS_FROM_DOM_ELEMENTS_TABLE
+				// TODO: look the dtype up in the volume metadata
+				var texture = new THREE.Texture3D(volume.data, volume.xLength, volume.yLength, volume.zLength, THREE.RedFormat, THREE.FloatType);
```

Comment:
We've updated `Texture3D` constructor (see #14861).
Could you update this line?